src/algorithm/hnswlib/StreamWriter.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024-present the vsag project
```

Comment:
code style: add a blank line before this

Reply:
add a blank line at start of file for all files